app/Models/Message.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Gladiator\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Message extends Model
+{
+    /**
+     * Array of available message types.
+     *
+     * @var array
+     */
+    protected static $types = [
```

Comment:
Cool. Yeah, nice thing with doing this as an array on the model it's easy to update. I was going off of the **Gladiator - Messaging** list of template Sr. Harbur set up.